lib/api.js

```diff
@@ -0,0 +1,39 @@
+"use strict";
```

Comment:
We shouldn't store `lib` files in the repo, since they are compiler-generated executable files, not source files. We store only sources, resources and metadata in the repo.